test/DotNetEnv.Tests/ParserTests.cs

```diff
@@ -361,6 +386,21 @@ public void ParseAssignment ()
             Assert.Throws<ParseException>(() => Parsers.Assignment.End().Parse("EV_DNE='"));
             Assert.Throws<ParseException>(() => Parsers.Assignment.End().Parse("EV_DNE=0\n1"));
 
+            testParse("EV_DNE", "a'b'' 'c' d", "EV_DNE=\"a'b'' 'c' d\" #allow inline singleQuotes in doubleQuoted values");
```

Comment:
I don't want to call these "inline" when they are inside a (single/double) quoted value. So, aside from the terminology, I think these are good additions.